openwisp_users/accounts/adapter.py

```diff
@@ -0,0 +1,16 @@
+from allauth.account.adapter import DefaultAccountAdapter
+from django.utils.translation import gettext as _
+from openwisp_utils.admin_theme import settings as utils_settings
+
+
+class EmailAdapter(DefaultAccountAdapter):
+    def send_mail(self, template_prefix, email, context):
+        context["logo_url"] = utils_settings.OPENWISP_EMAIL_LOGO
+        site_name = context["current_site"].name
+        context["subject"] = _("Welcome to %(site_name)s") % {"site_name": site_name}
```

Comment:
please ensure quoting style is consistent with the rest of the codebase (single quotes).